src/main/java/data/SheetsConverter.java

```diff
@@ -69,34 +69,42 @@ public void clearSheets(Sheets sheetsService) throws IOException {
       .clear(SPREADSHEET_ID, TAB_2, requestBody).execute();
   } 
 
-  /** Gets every Account that is associated with any Vendor from the List. */
-  public ArrayList<ArrayList<Account>> getAllAccounts(ArrayList<Vendor> vendors) {
-    ArrayList<ArrayList<Account>> allAccounts =  new ArrayList<ArrayList<Account>>();
+  /** Store the Accounts(value) associated to its Vendor(key) in a HashMap. */
+  public HashMap<String, ArrayList<Account>> getAllAccounts(
+    ArrayList<Vendor> vendors) {      
+
```

Comment:
can you indent these lines for better readability?

Reply:
Sure.